pinot-common/src/main/java/org/apache/pinot/common/utils/fetcher/SegmentFetcherFactory.java

```diff
@@ -153,6 +153,16 @@ private void fetchSegmentToLocalInternal(URI uri, File dest)
     getSegmentFetcher(uri.getScheme()).fetchSegmentToLocal(uri, dest);
   }
 
+  public static File fetchUntarToLocalStreamed(String uri, File tempRootDir, long rateLimit)
```

Comment:
(nit) `fetchAndStreamUntarSegmentToLocal` ? to be consistent with how other methods are named

- fetchSegmentToLocal
- fetchAndDecryptSegmentToLocal

Reply:
Add 1-2 lines of javadoc please

Reply:
done